user-access-handlers/src/test/java/features/RoleFt.java

```diff
@@ -31,7 +31,7 @@ public RoleFt(ScenarioContext scenarioContext) {
         super(scenarioContext);
     }
 
-    @Given("^an (\\w*) with role-name \"(\\w*)\" that exists in the Database$")
+    @Given("^an{0,1} (\\w*) with role-name \"(\\w*)\" that exists in the Database$")
```

Comment:
Fixed regex for feature tests. In next PR feature tests will be deleted, because they were a bad idea. Unit and Integration tests have proven enough for providing an overview of what a handler does.